src/xrEngine/device.cpp

```diff
@@ -34,6 +34,7 @@ ENGINE_API BOOL g_bRendering = FALSE;
 
 BOOL g_bLoaded = FALSE;
 ref_light precache_light = 0;
+int g_dwFPSlimit = 120;
```

Comment:
Leave your game in main menu a while in vanilla CoP. Trust me, you will hear coil whine because it renders 2000 FPS.